core/shared/src/main/scala/cats/effect/IO.scala

```diff
@@ -23,10 +23,16 @@ import scala.annotation.unchecked.uncheckedVariance
 import scala.concurrent.ExecutionContext
 import scala.concurrent.duration._
 
-sealed abstract class IO[+A] private () extends IOPlatform[A] {
+sealed abstract class IO[+A] private () {
 
   private[effect] def tag: Byte
 
+  def *>[B](that: IO[B]): IO[B] =
+    flatMap(_ => that)
+
+  def >>[B](that: => IO[B]): IO[B] =
+    flatMap(_ => that)
+
```

Comment:
👍 Though, for uniformity, I would also like:

- `<*`
- `productR`
- `productL`

I think we should try to follow cats as closely as possible on this one.